types/utils.go

```diff
@@ -1,6 +1,21 @@
 package types
 
-import "reflect"
+import (
+	"reflect"
+	"time"
+)
+
+type StepDuration struct {
```

Comment:
I think that the difference between `start` and `end` was simply recorded before. How should this structure be used?

Reply:
This structure has already been in use. The one in `consensus/state.go` has been moved to `types` for use in `state` packages. But I found that the one in `state.go` hasn't been removed. I'll remove it when I apply the modifications.

Reply:
Sorry, as you said, it was my misunderstanding. I made a mistake in the place to comment. Thank you for your explanation.